bgpd/bgp_route.c

```diff
@@ -9917,17 +9917,13 @@ DEFUN (show_ip_bgp_route,
 			      path_type, uj);
 }
 
-DEFUN (show_ip_bgp_regexp,
```

Comment:
Whitespace chages?

Reply:
Yeah please put these back the way they were. `clang-format` mangles DEFUNs, can't run it on those

Reply:
Yeah, I change the single line only, but clang-format scatters the whole block :)